source/_components/sensor.markdown

```diff
@@ -9,8 +9,15 @@ sharing: true
 footer: true
 ---
 
-Sensors are gathering information about states and conditions. 
+Sensors are gathering information about states and conditions.
 
 Home Assistant currently supports a wide range of sensors. They are able to display information which are provides by Home Assistant directly, are gathered from web services, and, of course, physical devices. Please check the sidebar for a full list of supported sensor platforms.
 
+### {% linkable_title Device Class %}
```

Comment:
The configuration variables generator is pointing `device_class` options to `/components/%{component}/#device_class`, so there needs to be a title here for the link to work.

https://github.com/home-assistant/home-assistant.github.io/blob/7bfe31cbac81b23764066ed41a8d988384a78f01/plugins/configuration.rb#L5